pages/applications/add-case.tsx

```diff
@@ -55,12 +57,22 @@ export default function AddCasePage({ user }: PageProps): JSX.Element {
       assignedTo: user.email,
     };
 
-    createApplication(request).then(() => {
-      router.push({
-        pathname: '/applications/view-register',
-        query: { status: ApplicationStatus.MANUAL_DRAFT },
+    createManualApplication(request);
+  };
+
+  const createManualApplication = async (request: Application) => {
+    try {
+      const newApplication = await createApplication(request);
+      const completedApplication = await completeApplication(newApplication);
+      const setToManualDraft = await updateApplication({
+        ...completedApplication,
+        status: ApplicationStatus.MANUAL_DRAFT,
       });
-    });
+
+      router.push(`/applications/view/${setToManualDraft.id}`);
+    } catch (error) {
+      console.error(error);
```

Comment:
What's the benefit of catching this error like this?

Reply:
Will remove this.